packages/swapper/src/swappercli.ts

```diff
@@ -89,6 +95,8 @@ const main = async (): Promise<void> => {
     rpcUrl: 'https://mainnet.infura.io/v3/d734c7eebcdf400185d7eb67322a7e57',
   })
 
+  const bip44Params = ethChainAdapter.buildBIP44Params({ accountNumber })
```

Comment:
should use `ethChainAdapter.getBIP44Params({acctNum})`. need to remove `build` or rename `get` to `build`